tutor/src/helpers/time.coffee

```diff
@@ -14,6 +13,7 @@ TIME_LINKS =
   'US/Eastern': 'Eastern Time (US & Canada)'
   'US/East-Indiana': 'Indiana (East)'
   'Canada/Atlantic': 'Atlantic Time (Canada)'
+  'Asia/Shanghai': 'Shanghai'
 
```

Comment:
These populate the dropdown on the course settings. Do we want to add 'Asia/Shanghai' to them, or was this a debug thing that got left in ?